ts/twitter/src/lib/sdk/rabbitmq-logger-sdk/rabbitmq-logger.ts

```diff
@@ -2,7 +2,19 @@ import amqp from "amqplib";
 
 export class RabbitMqLogger {
   private static readonly QUEUE_NAME = "logs";
-  async sendLog(eventName: string, objectId: string): Promise<void> {
+
+  async sendTapLog(objectId: string): Promise<void> {
+    await this.sendLogMessage("tap", objectId);
+  }
+
+  async sendImpressionLog(objectId: string): Promise<void> {
+    await this.sendLogMessage("impression", objectId);
+  }
+
+  private async sendLogMessage(
```

Comment:
```suggestion
  private async sendLog(
```
I think the original function name, `sendLog`, is sufficient since this function doesn't send the message.